recipes/r-cmls/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '1.0-0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-cmls
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/CMLS_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/CMLS/CMLS_{{ version }}.tar.gz
+  sha256: 765f5d122fb37b93ac121bb41989af6b4260442bc55ac82fcb7a6a4335c68294
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```

Comment:
This one should be noarch and the `{{posix}}zip               # [win]` should be removed.